perfkitbenchmarker/vm_util.py

```diff
@@ -238,42 +240,55 @@ def RunThreaded(target, thread_params, max_concurrent_threads=200):
     return
 
   if not isinstance(thread_params[0], tuple):
-    thread_params = [((arg,), {}) for arg in thread_params]
+    target_arg_tuples = [(target, (arg,), {}) for arg in thread_params]
```

Comment:
Consider moving the logic below into a function that accepts `target_arg_tuples`, so multiple functions can be run concurrently as in the multiprocess case.